pages/ksphere/konvoy/latest/tutorials/replacing-failed-control-plane-node/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: layout.pug
+navigationTitle: Replacing a failed control plane node 
+title: Replacing a failed control plane node 
+excerpt: Learn how to deploy a sample application on the Konvoy cluster
+menuWeight: 5
+---
+
+# Replacing a failed control plane node
+
+<p class="message--note"><strong>NOTE: </strong>If a majority of control plane nodes failed permanently, for example the EC2 instance has been terminated, then a new etcd cluster must be created.</p>
+
+1. In the AWS console terminate the failed control plane node.
+
+1. Exec into a `working` control plane node.
+```shell
+kubectl exec -ti -n kube-system etcd-ip-10-0-193-25.us-west-2.compute.internal sh
+```
+
+1. From the `control plane shell` run the following command to list all the etcd members, and note down the `member id` of the failing control plane node, in this sample `adcebc0f12cd5b7f`.
```

Comment:
Does the bullet ordering come out right? In markdown view the numbers keep reseting

![Screen Shot 2020-02-27 at 2 12 01 PM](https://user-images.githubusercontent.com/4225547/75491698-34c63880-596b-11ea-9fb0-9dc908dddfda.png)